appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectToolbar.java

```diff
@@ -157,7 +157,8 @@ public void onSuccess(Void result) {
               Ode.getInstance().getProjectManager().removeProject(projectId);
               // Show a welcome dialog in case there are no
               // projects saved.
-              if (Ode.getInstance().getProjectManager().getProjects().size() == 0) {
+              if (Ode.getInstance().getProjectManager().getProjects().size() == 0
+                  && Ode.getInstance().getCurrentView() != 2) { //2 is the int for Gallery
```

Comment:
I agree, it's a silly thing. In Ode.java (starting on line 181) it has this code:
```
  // Remembers the current View
  static final int DESIGNER = 0;
  static final int PROJECTS = 1;
  private static final int GALLERY = 2;
  private static final int GALLERYAPP = 3;
  private static final int USERPROFILE = 4;
  private static final int PRIVATEUSERPROFILE = 5;
  private static final int MODERATIONPAGE = 6;
  private static int currentView = DESIGNER;
```
I'm not sure how to extract this to a constant that exists outside of Ode. Where do those live?